packages/node_modules/nav-frontend-js-utils/src/eventThrottler.js

```diff
@@ -0,0 +1,40 @@
+import PT from 'prop-types';
+import { Component } from 'react';
+import _throttle from 'lodash.throttle';
+
+export class EventThrottler extends Component { // eslint-disable-line import/prefer-default-export
+    componentDidMount() {
+        window.addEventListener(this.props.event, this.throttleFn()());
+    }
+
+    componentWillUnmount() {
+        if (this.actualThrottleFn) {
+            window.removeEventListener(this.props.event, this.actualThrottleFn);
+        }
+    }
+
+    throttleFn() {
+        let cb = this.props.callback;
+        cb = cb.bind(this.props.context); // eslint-disable-line react/prop-types
```

Comment:
Må vi gjøre binding av context? Kan vi ikke bare anta at folk dealer med dette selv? Og hvorfor er den ikke som en del av proptypes?

Reply:
Jeg tok med binding av context for å slippe å måtte legge på det på komponentnivå hver gang man trenger å bruke EventThrottler med en bestemt kontekst - det ga mening der og da. Tror du det er bedre å droppe det helt?

Tok ikke med i PT siden jeg ikke fikk lov av eslint til å bruke object som proptype, men den bortfaller uansett helt hvis vi skal fjerne kontekst-bindinga.